crates/oxc_minifier/src/lib.rs

```diff
@@ -42,6 +42,7 @@ impl Minifier {
 
     pub fn build<'a>(self, allocator: &'a Allocator, program: &mut Program<'a>) -> MinifierReturn {
         Compressor::new(allocator, self.options.compress).build(program);
+        // TODO: pass top_level to mangler
```

Comment:
How should I expose this option in the minifier?
Should I change `pub mangle: bool` in the MinifierOptions to `pub mangle: MangleOptions`?

Reply:
`pub mangle: MangleOptions`

Reply:
I changed to `Option<MangleOptions>` so that `mangle: None` can be done to disable the mangler.